core/survey/_surveyValidator/surveyInfoValidator.js

```diff
@@ -9,6 +9,43 @@ const validateSurveyNameUniqueness = surveyInfos => (propName, survey) => {
     : null
 }
 
+const validateCycles = (propName, nodeDef) => {
```

Comment:
the error should be associated to each of the invalid cycle.
the error could be something like ```{
fields: {
   '0':{
     valid : false,
     fields: {}
}
}
}```

to create the error result you can use ```Validation.newInstance``` 

Plus, add to the UI a ```<ValidationTooltip/>``` around the invalid cycle

Reply:
As far as I can tell, currently these validations only run when publishing the survey.
Should there be the same validations on the UI fields also when editing the survey info? (and not just when publishing it)

Reply:
the survey info validation is also applied when saving the infos from the UI. that's where the error should popup. see survey api ```app.put('/survey/:surveyId/info')```->>```SurveyManager.updateSurveyProps```